packages/fxa-auth-server/lib/payments/stripe.ts

```diff
@@ -2081,35 +2078,24 @@ export class StripeHelper extends StripeHelperBase {
       }
     }
     if (customer.default_source) {
-      if (typeof customer.default_source !== 'string') {
-        const { brand, exp_month, exp_year, funding, last4, name } =
-          customer.default_source as Stripe.Card;
-        return {
-          billing_name: name,
-          payment_provider: paymentProvider,
-          payment_type: funding,
-          last4,
-          exp_month,
-          exp_year,
-          brand,
-        };
-      } else {
-        const paymentMethod = await this.expandResource<Stripe.PaymentMethod>(
-          customer.default_source,
-          PAYMENT_METHOD_RESOURCE
-        );
-        const { brand, exp_month, exp_year, funding, last4 } =
-          paymentMethod.card!;
-        return {
-          billing_name: customer.name,
-          payment_provider: paymentProvider,
-          payment_type: funding,
-          last4,
-          exp_month,
-          exp_year,
-          brand,
-        };
-      }
+      const paymentMethod = await this.expandResource<Stripe.PaymentMethod>(
```

Comment:
I'm not sure why that method is doing the test, since the `expandResource` will just return it if its not a string.

Reply:
`expandResource` would return it, but then it would look for `brand`, `exp_month`, etc within `paymentMethod.card`, which I don't think exists in `customer.default_source`?

Reply:
Ben's saying that if you pass it a source object (not a string), `expandResource` would just return that same source object (because it's not a string).